composer.json

```diff
@@ -11,12 +11,12 @@
         }
     ],
     "require": {
-        "php": "^7.1",
-        "moontoast/math": "^1.1",
-        "ramsey/uuid": "^3.0"
+        "php": "^7.2",
+        "brick/math": "^0.8",
+        "ramsey/uuid": "^4.0"
```

Comment:
I don't think there's any reason to force downstream users to upgrade to ramsey/uuid 4, unless this library uses features only available in version 4, and I don't think it does.